components/Chat/Chat.vue

```diff
@@ -226,6 +259,16 @@
           console.warn('Could not find scroll container for chat.');
       },
 
+      async ignoreUser ( user ) {
+        const actions = await this.$chatCommandParser.ignoreUser( user );
+        actions?.forEach( a => this.executeAction( a ) );
```

Comment:
wait, if the `variable?.property` syntax works here (I thought it needed typescript) we can simplify code in a lot of other places lol. I'll have to check that.